tableau_utilities/scripts/csv_config.py

```diff
@@ -33,7 +33,10 @@ def csv_config(args):
             for datasource in details['datasources']:
                 datasource_name = datasource['name']
                 local_name = datasource['local-name']
-                sql_alias = datasource['sql_alias']
+
+                sql_alias = None
+                if sql_alias in datasource:
+                    sql_alias = datasource['sql_alias']
```

Comment:
This can be simplified
```suggestion
                sql_alias = datasource.get('sql_alias')
```
But if you want to keep the `if` logic, it should be
`if 'sql_alias' in datasource`

Reply:
@JustinGrilli changed a few of them to `.get`.  Thanks!